pkg/config/templates/rancherd-22-addons.yaml

```diff
@@ -24,9 +28,176 @@ resources:
       repo: http://harvester-cluster-repo.cattle-system.svc/charts
       version: "0.2.4-rc2"
       chart: harvester-pcidevices-controller
+      {{- if and .Addons .Addons.harvester_pcidevices_controller }}
+      enabled: {{ .Addons.harvester_pcidevices_controller.Enabled }}
+      {{- else }}
       enabled: false
+      {{- end }}
       valuesContent: |
         image:
           tag: v0.2.4-rc2
         fullnameOverride: harvester-pcidevices-controller
-
+  - apiVersion: harvesterhci.io/v1beta1
+    kind: Addon
+    metadata:
+      name: rancher-logging
+      namespace: cattle-logging-system
+    spec:
+      repo: http://harvester-cluster-repo.cattle-system.svc/charts
+      version: "100.1.3+up3.17.7"
```

Comment:
Should we bump to the newer chart version https://github.com/rancher/charts/blob/dev-v2.7/charts/rancher-logging/102.0.0%2Bup3.17.10? so that it is more compatible with Rancher v2.7 & has the newer version of fluentbit.

Reply:
The most important change between `100.1.3+up3.17.7` and `102.0.0+up3.17.10` is as below, need to further check the affect to upgrade.

![image](https://user-images.githubusercontent.com/31133476/231130878-bed77bf6-8fe1-4c80-be63-ecce9a6268ee.png)

![image](https://user-images.githubusercontent.com/31133476/231130985-aaafb747-400a-4217-8c2e-a8612149220e.png)